src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  width: $card-width;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    display: flex;
+    padding-top: 40px;
+    padding-bottom: 4px;
+
+    &-text {
+      margin: 0;
+      font-weight: 500;
+      font-size: $card-name-font-size;
+      line-height: 18px;
+      color: $color-primary;
+    }
+  }
+
+  &__product-code {
+    display: flex;
+    font-weight: 400;
+    font-size: $secondary-font-size;
+    line-height: 14px;
+    color: $color-secondary;
+    width: $card-width;
+  }
+
+  &__reviews {
+    display: flex;
+    align-self: center;
+    font-size: $secondary-font-size;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-primary;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: $card-width;
+    padding-top: 24px;
+
+    &-label {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $color-secondary;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      color: $color-primary;
+    }
+  }
+
+  &__button {
+    padding-top: 16px;
+  }
+
+  .button {
```

Comment:
The `.button` class is nested within the `.card` class, which might lead to specificity issues. Consider using a more specific class name or moving it outside of the `.card` block if it's intended to be a global button style.